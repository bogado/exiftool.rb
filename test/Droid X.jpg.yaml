---
:source_file: /Users/mrm/Dropbox/code/exiftoolr/test/Droid X.jpg
:exif_tool_version: 9.31
:file_name: Droid X.jpg
:directory: /Users/mrm/Dropbox/code/exiftoolr/test
:file_size: 2023 kB
:file_modify_date: 2012-02-22 21:22:17.000000000 -08:00
:file_access_date: 2013-07-14 11:51:21.000000000 -07:00
:file_inode_change_date: 2013-02-16 11:21:58.000000000 -08:00
:file_permissions: rwxr-xr-x
:file_type: JPEG
:mime_type: image/jpeg
:exif_byte_order: Little-endian (Intel, II)
:make: Motorola
:model: DROIDX 6d460001fff80000015d7d330e01c011
:orientation: Rotate 90 CW
:x_resolution: 300
:y_resolution: 300
:resolution_unit: inches
:software: 2.2
:modify_date: 2010-12-08 01:15:37.000000000 -08:00
:y_cb_cr_positioning: Centered
:exposure_time: !ruby/object:Rational
  denominator: 15
  numerator: 1
:f_number: 2.8
:exposure_program: Program AE
:iso: 637
:exif_version: '0220'
:date_time_original: 2010-12-08 01:15:37.000000000 -08:00
:create_date: 2010-12-08 01:15:37.000000000 -08:00
:components_configuration: Cb, Y, Cr, Y
:compressed_bits_per_pixel: 4
:shutter_speed_value: !ruby/object:Rational
  denominator: 15
  numerator: 1
:aperture_value: 2.8
:brightness_value: -0.8203125
:exposure_compensation: 0
:max_aperture_value: 2.8
:subject_distance: 0 m
:metering_mode: Center-weighted average
:light_source: Standard Light A
:flash: Auto, Did not fire
:focal_length: 5.0 mm
:user_comment: ''
:flashpix_version: '0100'
:color_space: sRGB
:exif_image_width: 3264
:exif_image_height: 1840
:interop_index: R98 - DCF basic file (sRGB)
:interop_version: '0100'
:exposure_index: 637
:sensing_method: One-chip color area
:file_source: Digital Camera
:scene_type: Directly photographed
:custom_rendered: Custom
:exposure_mode: Auto
:white_balance: Auto
:digital_zoom_ratio: 0
:focal_length_in35mm_format: 38 mm
:scene_capture_type: Standard
:gain_control: High gain up
:contrast: Normal
:saturation: Normal
:sharpness: Normal
:subject_distance_range: Unknown
:image_unique_id: ''
:compression: JPEG (old-style)
:thumbnail_offset: 2150
:thumbnail_length: 32586
:image_width: 3264
:image_height: 1840
:encoding_process: Baseline DCT, Huffman coding
:bits_per_sample: 8
:color_components: 3
:y_cb_cr_sub_sampling: YCbCr4:2:0 (2 2)
:aperture: 2.8
:image_size: 3264x1840
:scale_factor35efl: 7.6
:shutter_speed: !ruby/object:Rational
  denominator: 15
  numerator: 1
:thumbnail_image: (Binary data 32586 bytes)
:circle_of_confusion: 0.004 mm
:fov: 50.7 deg
:focal_length35efl: '5.0 mm (35 mm equivalent: 38.0 mm)'
:hyperfocal_distance: 2.24 m
:light_value: 4.2
