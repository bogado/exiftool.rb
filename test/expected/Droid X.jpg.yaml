---
:file_name: Droid X.jpg
:file_size: 2071450
:file_type: JPEG
:file_type_extension: JPG
:mime_type: image/jpeg
:exif_byte_order: II
:make: Motorola
:model: DROIDX 6d460001fff80000015d7d330e01c011
:orientation: 6
:x_resolution: 300
:y_resolution: 300
:resolution_unit: 2
:software: 2.2
:modify_date: 2010:12:08 01:15:37
:modify_date_civil: 2010-12-08
:y_cb_cr_positioning: 1
:exposure_time: 0.066667
:f_number: 2.8
:exposure_program: 2
:iso: 637
:exif_version: '0220'
:date_time_original: 2010:12:08 01:15:37
:date_time_original_civil: 2010-12-08
:create_date: 2010:12:08 01:15:37
:create_date_civil: 2010-12-08
:components_configuration: 2 1 3 1
:compressed_bits_per_pixel: 4
:shutter_speed_value: 0.0666962750423015
:aperture_value: 2.79795934507662
:brightness_value: -0.8203125
:exposure_compensation: 0
:max_aperture_value: 2.79795934507662
:subject_distance: 0
:metering_mode: 2
:light_source: 17
:flash: 24
:focal_length: 4.97
:user_comment: ''
:flashpix_version: '0100'
:color_space: 1
:exif_image_width: 3264
:exif_image_height: 1840
:interop_index: R98
:interop_version: '0100'
:exposure_index: 637
:sensing_method: 2
:file_source: 3
:scene_type: 1
:custom_rendered: 1
:exposure_mode: 0
:white_balance: 0
:digital_zoom_ratio: 0
:focal_length_in35mm_format: 38
:scene_capture_type: 0
:gain_control: 2
:contrast: 0
:saturation: 0
:sharpness: 0
:subject_distance_range: 0
:image_unique_id: ''
:compression: 6
:thumbnail_offset: 2150
:thumbnail_length: 32586
:image_width: 3264
:image_height: 1840
:encoding_process: 0
:bits_per_sample: 8
:color_components: 3
:y_cb_cr_sub_sampling: 2 2
:aperture: 2.8
:image_size: 3264x1840
:megapixels: 6.00576
:shutter_speed: 0.066667
:light_value: 4.20644366401025
:fov: 50.6923947019274
:hyperfocal_distance: 2.2448716941235
